schema.sql

```diff
@@ -9,3 +9,7 @@ CREATE TABLE animals(
   weight_kg        DECIMAL 
 );
 
+ALTER TABLE animals
+ADD species TEXT;
```

Comment:
- [ ] Kindly note the datatype is varchar, this is to have more control over the character limit